silver/api/serializers.py

```diff
@@ -462,3 +462,27 @@ def update(self, instance, validated_data):
                 })
 
         return super(PaymentSerializer, self).update(instance, validated_data)
+
+
+class PaymentProcessorUrl(serializers.HyperlinkedRelatedField):
+    def __init__(self, view_name=None, **kwargs):
+        assert view_name is not None, 'The `view_name` argument is required.'
```

Comment:
`HyperlinkedRelatedField` already takes care of this assertion in its own `__init__`. Isn't it safe to wait for the parent to crash 3 lines below?